primitives/rpc/src/tracing.rs

```diff
@@ -35,6 +35,9 @@ pub struct BlockTrace {
 	/// Storage key targets used to filter out events that do not have one of the storage keys.
 	/// Empty string means do not filter out any events.
 	pub storage_keys: String,
+	/// Method targets used to filter out events that do not have one of the event method.
+	/// Empty string means do not filter out any events.
```

Comment:
I wonder if this empty string business is going to come back and bite us one day. Ideally we'd have an `Option<String>` here, but I understand why you went with the same we had before.